test/rekt/features/eventtransform/eventtransform.go

```diff
@@ -375,12 +379,13 @@ func JsonataSinkTLS() *feature.Feature {
 	})
 
 	f.Prerequisite("transport encryption is strict", featureflags.TransportEncryptionStrict())
-	f.Prerequisite("transport encryption is strict", featureflags.AuthenticationOIDCEnabled())
 	f.Prerequisite("should not run when Istio is enabled", featureflags.IstioDisabled())
 
-	f.Setup("Install event transform", eventtransform.Install(transformName, eventtransform.WithSpec(
-		eventtransform.WithSink(&duckv1.Destination{URI: apis.HTTPS(sink)}),
-		eventtransform.WithJsonata(eventing.JsonataEventTransformationSpec{Expression: `
+	f.Setup("Install event transform", func(ctx context.Context, t feature.T) {
+
+		eventtransform.Install(transformName, eventtransform.WithSpec(
+			eventtransform.WithSink(&duckv1.Destination{URI: apis.HTTPS(fmt.Sprintf("%s.%s.svc.%s", sink, environment.FromContext(ctx).Namespace(), network.GetClusterDomainName()))}),
```

Comment:
why does it need a different URI?

Reply:
because of the subject alt names of the certificate